packages/web-app-files/src/components/SideBar/TagsPanel.vue

```diff
@@ -11,6 +11,7 @@
         :options="availableTags"
         taggable
         push-tags
+        :select-on-key-codes="[188, 13]"
```

Comment:
No magic numbers please :-)

Reply:
We already have the dependency `keycode` (see root level package.json). You can use that to create an array with human readable keys in the setup function, return that from the setup function, and use it here in the template. (e.g. `keycode('enter')` returns `13`, you can just throw that into an array.